src/EditorFeatures/CSharp/Formatting/CSharpEditorFormattingService.cs

```diff
@@ -97,9 +97,11 @@ public bool SupportsFormattingOnTypedCharacter(Document document, char ch)
 
             var span = textSpan ?? new TextSpan(0, root.FullSpan.Length);
             var formattingSpan = CommonFormattingHelpers.GetFormattingSpan(root, span);
-            return Formatter.GetFormattedTextChanges(root, 
+#pragma warning disable VSTHRD103 // Call async methods when in an async method
+            return Formatter.GetFormattedTextChanges(root,
```

Comment:
@heejaechang should this one get fixed?

Reply:
didn't know formatter became async. anyway yes. looks like we should use async version.